update/ansible/playbook/tasks/roles/sqlite-to-postgres/tasks/main.yml

```
@@ -1,4 +1,12 @@
 ---
+- name: Wait for grafana to update DB and be ready
```

```suggestion
- name: Wait for grafana to finish up the upgrade
```

I believe grafana is finishing up the upgrade, which is larger then a database update.

---

In this case, we need to wait for the DB upgrade to migrate the upgraded DB to PostgreSQL.